develop: TestGlCanvas_walkTree fails

Since f5332ba57a95c042f95257f539a87bccfc030829 (maybe 10a773dc008c7961136ecf32c86717fd6a08bdfb) #1801 TestGlCanvas_walkTree fails with:

```
=== RUN   TestGlCanvas_walkTree
    canvas_test.go:574: 
        	Error Trace:	canvas_test.go:574
        	Error:      	Not equal: 
        	            	expected: []glfw.beforeCall{glfw.beforeCall{obj:(*fyne.Container)(0xc000020cc0), parent:fyne.CanvasObject(nil), pos:fyne.Position{X:17, Y:42}}, glfw.beforeCall{obj:(*fyne.Container)(0xc000020c40), parent:(*fyne.Container)(0xc000020cc0), pos:fyne.Position{X:317, Y:442}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306840), parent:(*fyne.Container)(0xc000020c40), pos:fyne.Position{X:318, Y:444}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc0003068a0), parent:(*fyne.Container)(0xc000020c40), pos:fyne.Position{X:337, Y:472}}, glfw.beforeCall{obj:(*fyne.Container)(0xc000020c80), parent:(*fyne.Container)(0xc000020cc0), pos:fyne.Position{X:24, Y:50}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306900), parent:(*fyne.Container)(0xc000020c80), pos:fyne.Position{X:524, Y:650}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306960), parent:(*fyne.Container)(0xc000020c80), pos:fyne.Position{X:84, Y:120}}}
        	            	actual  : []glfw.beforeCall{glfw.beforeCall{obj:(*fyne.Container)(0xc000020cc0), parent:fyne.CanvasObject(nil), pos:fyne.Position{X:17, Y:42}}, glfw.beforeCall{obj:(*fyne.Container)(0xc000020c40), parent:(*fyne.Container)(0xc000020cc0), pos:fyne.Position{X:317, Y:442}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306840), parent:(*fyne.Container)(0xc000020c40), pos:fyne.Position{X:318, Y:444}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc0003068a0), parent:(*fyne.Container)(0xc000020c40), pos:fyne.Position{X:337, Y:472}}, glfw.beforeCall{obj:(*fyne.Container)(0xc000020c80), parent:(*fyne.Container)(0xc000020cc0), pos:fyne.Position{X:24, Y:50}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306900), parent:(*fyne.Container)(0xc000020c80), pos:fyne.Position{X:24, Y:50}}, glfw.beforeCall{obj:(*canvas.Rectangle)(0xc000306960), parent:(*fyne.Container)(0xc000020c80), pos:fyne.Position{X:24, Y:55}}}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1052,4 +1052,4 @@
        	            	   pos: (fyne.Position) {
        	            	-   X: (float32) 524,
        	            	-   Y: (float32) 650
        	            	+   X: (float32) 24,
        	            	+   Y: (float32) 50
        	            	   }
        	            	@@ -1170,4 +1170,4 @@
        	            	   pos: (fyne.Position) {
        	            	-   X: (float32) 84,
        	            	-   Y: (float32) 120
        	            	+   X: (float32) 24,
        	            	+   Y: (float32) 55
        	            	   }
        	Test:       	TestGlCanvas_walkTree
        	Messages:   	calls before children hook with the correct node and position
--- FAIL: TestGlCanvas_walkTree (0.00s)
```

**Note**: it might be necessary to disable the loop’s window closed handling (lines 103-148 of glfw/loop.go) to ensure that the test runs do not exit prematurely.

--- a/internal/driver/glfw/canvas_test.go
+++ b/internal/driver/glfw/canvas_test.go
@@ -498,10 +498,10 @@ func TestGlCanvas_SetContent(t *testing.T) {
 func TestGlCanvas_walkTree(t *testing.T) {
 	leftObj1 := canvas.NewRectangle(color.Gray16{Y: 1})
 	leftObj2 := canvas.NewRectangle(color.Gray16{Y: 2})
-	leftCol := container.NewVBox(leftObj1, leftObj2)
+	leftCol := container.NewWithoutLayout(leftObj1, leftObj2)
 	rightObj1 := canvas.NewRectangle(color.Gray16{Y: 10})
 	rightObj2 := canvas.NewRectangle(color.Gray16{Y: 20})
-	rightCol := container.NewVBox(rightObj1, rightObj2)
+	rightCol := container.NewWithoutLayout(rightObj1, rightObj2)
 	content := container.NewWithoutLayout(leftCol, rightCol)
 	content.Move(fyne.NewPos(17, 42))
 	leftCol.Move(fyne.NewPos(300, 400))

